src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsApiMockServer.kt

```diff
@@ -77,7 +77,7 @@ class IncidentsApiMockServer : WireMockServer(WIREMOCK_PORT) {
     fun dpsIncidentReport(nomisIncidentId: String = "1234") = ReportWithDetails(
       id = UUID.randomUUID(),
       reportReference = nomisIncidentId,
-      type = ReportWithDetails.Type.ATTEMPTED_ESCAPE_FROM_ESCORT,
```

Comment:
hmmmm - could've just added my change to the automatically created PR

Reply:
sorry about this mega rename! i didn't think it'd affect you as the payload we accept for the sync is unchanged :(